src/dctest/util.cljs

```diff
@@ -61,8 +62,25 @@
          " " (:message error)
          (if (not (empty? params)) (str " " params) ""))))
 
+(defn validate-expression
+  [schema data]
+  (let [errors (expr/flatten-errors (expr/read-ast data schema))]
+    (if (empty? errors)
+      true
+      (do
+        (set! (.-errors validate-expression) (->js errors))
+        false))))
```

Comment:
Neat! Maybe a couple of comments about how this works and link to ajv docs on this?

Reply:
Neat is one word for it ;-) Good idea on the docs links.

Reply:
Yep, in my experience "neat" probably leans negative. And to be clear, I'm referring to underlying mechanism, not your decision to use it.